.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 8.1
+          php-version: 8.4
```

Comment:
Please don't change this, we want to keep generated clients compatible with 8.1, so testing needs to happen on 8.1